acropolis-service/src/main/kotlin/org/ephyra/acropolis/service/impl/ComputeInstanceService.kt

```diff
@@ -0,0 +1,29 @@
+package org.ephyra.acropolis.service.impl
+
+import org.ephyra.acropolis.persistence.api.entity.ComputeInstanceEntity
+import org.ephyra.acropolis.persistence.api.entity.ProjectEntity
+import org.ephyra.acropolis.persistence.api.persistence.ComputeInstancePersistence
+import org.ephyra.acropolis.service.api.IComputeInstanceService
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Component
+
+
+@Component
```

Comment:
use Service annotation rather than component here. Difference is superficial but makes the intent clear